packages/instrumentation/package.json

```diff
@@ -21,7 +24,7 @@
     "jest-junit": "16.0.0",
     "typescript": "5.4.5"
   },
-  "dependencies": {
+  "peerDependencies": {
     "@opentelemetry/api": "^1.8",
     "@opentelemetry/instrumentation": "^0.49 || ^0.50 || ^0.51 || ^0.52.0 || ^0.53.0",
     "@opentelemetry/sdk-trace-base": "^1.22"
```

Comment:
This one should just be gone, an instrumentation package must not depend on the SDK. The users likely won't even use `sdk-trace-base` directly but `sdk-trace-node` or even a third-party SDK not based on `sdk-trace-base`. In the latter case we may break completely because we'll be using a completely wrong `Span` class. Removing this depends on https://github.com/prisma/prisma/pull/25164.